templates/hooks/db_instance/sdk_read_many_post_set_output.go.tpl

```diff
@@ -1,10 +1,88 @@
+	// DescribeDBInstances returns an array of DBInstance structs that contains
+	// the *previously set* values for various mutable fields. This is
+	// problematic because it causes a "flopping" behaviour when the user has
+	// modified a Spec field from value A to value B but the output shape from
+	// ModifyDBInstance for that field contains value A, the standard SetOutput
+	// Go code generated above will set the Spec field to the *old* value
+	// again. The next time the reconciler runs, it will read the latest
+	// observed resource, see a difference between the desired and the latest
+	// state (that actually does not exist because the difference is comparing
+	// the value of the fields before they were changed) and attempt to modify
+	// the field from value B to value A again, causing a flop loop.
+	//
+	// Luckily, the Output shape's DBInstance struct contains a
+	// `PendingModifiedValues` struct which contains those field values that
+	// the user specified. So, we can use these to "reset" the Spec back to the
+	// appropriate user-specified values.
+	pmv := ko.Status.PendingModifiedValues
+	if pmv != nil {
+		if pmv.AllocatedStorage != nil {
+			ko.Spec.AllocatedStorage = pmv.AllocatedStorage
+		}
+		//if pmv.AutomationMode != nil {
```

Comment:
What's the logic behind skipping `AutomationMode` (as well as other fields, below)?

Could you add a comment here explaining why it isn't necessary to also make this copy?

Reply:
@jljaco those fields are included in later versions of aws-sdk-go. I will uncomment them in a followup PR when I update the aws-sdk-go in go.mod to a more recent version. I wanted to try and reduce the amount of changes in this PR and left these commented out to remind me to handle them for the newer aws-sdk-go version fields.

Reply:
Cool, understood!  Could you add a comment in the code to that effect?